elytron/src/main/resources/schema/wildfly-elytron_11_0.xsd

```diff
@@ -1164,6 +1165,33 @@
         </xs:complexContent>
     </xs:complexType>
 
+
+    <xs:complexType name="failoverRealmType">
+        <xs:annotation>
+            <xs:documentation>
+                A realm definition which wraps one realm and delegates to another in case the first is unavailable.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:complexContent>
+            <xs:extension base="realmType">
+                <xs:attribute name="delegate-realm" type="xs:string" use="required">
+                    <xs:annotation>
+                        <xs:documentation>
+                            The name of the realm to use as a default.
+                        </xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+                <xs:attribute name="failover-realm" type="xs:string" use="required">
+                    <xs:annotation>
+                        <xs:documentation>
+                            The name of the realm to use in case the default realm is unavailable.
+                        </xs:documentation>
+                    </xs:annotation>
```

Comment:
Looks like `emit-events` is missing here.

Reply:
fixed